rules/linux/lnx_process_discovery.yml

```diff
@@ -0,0 +1,23 @@
+title: Process Discovery
+id: 4e2f5868-08d4-413d-899f-dc2f1508627b
+status: stable
+description: Detects process discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1057/T1057.md
+logsource:
+    product: linux
+    category: process_creation
+detection:
+    selection:
+        - Image|endswith:
+          - '/ps'
```

Comment:
I think these are too generic for a detection, this causes far too much false positives.

Reply:
discussion here https://github.com/SigmaHQ/sigma/pull/1362#discussion_r598336458